controllers/admissionpolicy_controller.go

```diff
@@ -148,3 +156,41 @@ func (r *AdmissionPolicyReconciler) findAdmissionPoliciesForPolicyServer(ctx con
 	}
 	return r.findAdmissionPoliciesForConfigMap(&configMap)
 }
+
+func (r *AdmissionPolicyReconciler) findAdmissionPolicyForWebhookConfiguration(_ context.Context, webhookConfiguration client.Object) []reconcile.Request {
+	if _, found := webhookConfiguration.GetLabels()["kubewarden"]; !found {
+		return []reconcile.Request{}
+	}
+
+	policyScope, found := webhookConfiguration.GetLabels()["kubewardenPolicyScope"]
```

Comment:
I would say to add this string as a constant in the [constants.go](https://github.com/kubewarden/kubewarden-controller/blob/6361dc9a26885d4c14923665b57eba0b3a444121/internal/pkg/constants/constants.go#L39-L41). As we do for other labels as well. This applies for all the labels and annotations used in your changes

Reply:
done in: https://github.com/kubewarden/kubewarden-controller/pull/576/commits/7381c5d59ded0ec721cff4a18efe905db474774b